napalm_nxos/nxos.py

```diff
@@ -455,15 +455,27 @@ def get_bgp_neighbors(self):
             for neighbor_dict in neighbors_list:
                 neighborid = napalm_base.helpers.ip(neighbor_dict['neighbor-id'])
                 remoteas = napalm_base.helpers.as_number(neighbor_dict['remoteas'])
+                state = py23_compat.text_type(neighbor_dict['state'])
+
+                bgp_state_dict = {
```

Comment:
Maybe move this outside the for loop?  Not sure if there's a performance penalty, or if it's big enough to matter.